upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -211,7 +211,9 @@ func (tf *TemplateFunctions) DnsControllerArgv() ([]string, error) {
 	}
 	// permit wildcard updates
 	argv = append(argv, "--zone=*/*")
-	argv = append(argv, "-v=8")
+
+	// Verbose, but not crazy logging
+	argv = append(argv, "-v=2")
```

Comment:
It's so easy to change though if someone wants more logging.  Also I don't want to wait for the tests to rerun.  I'll push this with v=2 if that's OK, and then we can revise if needed.